### PR TITLE
chore(ui): tests for user wallet page

### PR DESCRIPTION
**Pull request by juliendonck**
Writing initial test for the user and org wallet. So far it only tests if the parts are there. 

Still missing:
- validation of fields (will need to wire up sending transactions first)
- testing of empty states
- testing of amounts

### CODE REVIEW

File: cypress/integration/org_wallet.spec.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    it("goes to receive tab and checks that everything is there", () => {
+      cy.pick("receive-tab").click();
+      cy.pick("receive").should("exist");
+      cy.pick("qr-code").should("exist");
+      cy.pick("urn").should("exist");
+    });
+    it("goes to back to send tab and checks that everything is there", () => {
+      cy.pick("send-tab").click();
+      cy.pick("send").should("exist");
+      // todo validate fields & test send functionality when implemented
```
Comment (MeBrei) [inline cypress/integration/org_wallet.spec.js:41]:
```suggestion
      // TODO(julien): validate fields & test send functionality when implemented
```

File: cypress/integration/org_wallet.spec.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    });
+  });
+  context("transactions", () => {
+    // checks the transactions component
+    it("has transaction component", () => {
+      // balance is present
```
Comment (MeBrei) [inline cypress/integration/org_wallet.spec.js:47]:
```suggestion
      // transactions are present
```

File: cypress/integration/user_wallet.spec.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    it("goes to receive tab and checks that everything is there", () => {
+      cy.pick("receive-tab").click();
+      cy.pick("receive").should("exist");
+      cy.pick("qr-code").should("exist");
+      cy.pick("urn").should("exist");
+    });
+    it("goes to back to send tab and checks that everything is there", () => {
+      cy.pick("send-tab").click();
+      cy.pick("send").should("exist");
+      // todo validate fields & test send functionality when implemented
```
Comment (MeBrei) [inline cypress/integration/user_wallet.spec.js:39]:
```suggestion
      // TODO(julien): validate fields & test send functionality when implemented
```

File: cypress/integration/user_wallet.spec.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    });
+  });
+  context("transactions", () => {
+    // checks the transactions component
+    it("has transaction component", () => {
+      // balance is present
```
Comment (MeBrei) [inline cypress/integration/user_wallet.spec.js:45]:
```suggestion
      // transactions are present
```

Comment (juliendonck) [reply cypress/integration/user_wallet.spec.js]: https://github.com/radicle-dev/radicle-upstream/pull/692/commits/406ff9348e2d94133db2feae7ad870579112462a

File: cypress/integration/user_wallet.spec.js (hunk 3 above)
Comment (juliendonck) [reply cypress/integration/user_wallet.spec.js]: https://github.com/radicle-dev/radicle-upstream/pull/692/commits/406ff9348e2d94133db2feae7ad870579112462a

File: cypress/integration/org_wallet.spec.js (hunk 2 above)
Comment (juliendonck) [reply cypress/integration/org_wallet.spec.js]: https://github.com/radicle-dev/radicle-upstream/pull/692/commits/406ff9348e2d94133db2feae7ad870579112462a

File: cypress/integration/org_wallet.spec.js (hunk 1 above)
Comment (juliendonck) [reply cypress/integration/org_wallet.spec.js]: https://github.com/radicle-dev/radicle-upstream/pull/692/commits/406ff9348e2d94133db2feae7ad870579112462a

File: ui/DesignSystem/Component/Wallet.svelte [hunk 5]
```diff
@@ -62,8 +62,11 @@
         <SendReceive {accountId} />
       </div>
     </div>
-    <div class="transactions" data-cy="transactions">
-      <Title variant="large" style="padding: 1.25rem 1.5rem;">
+    <div class="transactions">
```
Comment (juliendonck) [inline ui/DesignSystem/Component/Wallet.svelte]:
```suggestion
    <div class="transactions" data-cy="transactions">
```

**Pull request merged**